### PR TITLE
Formatting of pages and repo

### PR DESCRIPTION
**Pull request by Ki-er**
Added pages to /pages
Added github things to /.github

Formatted pages with two spaces between sections

### CODE REVIEW

File: Pages/Linux.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Supporting learning Linux
+
+- [Ask Ubuntu](https://askubuntu.com/) - Ubuntu Q&A site. Part of Stack Exchange.
+- [Unix & Linux Stack Exchange](https://unix.stackexchange.com/) - For more general Linux-related questions.
+
+
+## Linux Resources
+
+- [Arch Linux Wiki - Permissions](https://wiki.archlinux.org/title/File_permissions_and_attributes)-
```
Comment (CrimsonTome) [inline Pages/Linux.md]: can you remove the trailing `-` from here?

File: README.md [hunk 2]
```diff
@@ -14,18 +15,17 @@ List originally comprised by [@closebracket](https://github.com/closebracket)
...
 - [Other pages](#other-pages)
-  - [Cyber Security](/CyberSecurity.md)
-  - [Electronic](/Electronics.md)
-  - [Ethics, Law and Professionalism](/Ethics_Law_Professionalism.md)
-  - [Programming](/Programming.md)
-  - [Virtualisation](/Virtualisation.md)
+  - [Cyber Security](/Pages//CyberSecurity.md)
+  - [Electronic](/Pages//Electronics.md)
+  - [Ethics, Law and Professionalism](/Pages//Ethics_Law_Professionalism.md)
+  - [Linux](/Pages/Linux.md)
```
Comment (CrimsonTome) [inline README.md:23]: double `//`?

**Review by CrimsonTome — CHANGES_REQUESTED** [review]
see the comments made

**Comment by Ki-er** [comment]
Changes made

**Comment by CrimsonTome** [comment]
Thanks for the overhaul!

**Pull request merged**